src/modules/groups/groups.js

```diff
@@ -1,16 +1,14 @@
 const express = require('express');
 const axios = require('axios');
+
 const router = express.Router();
-// const validate = require('validate-azure-ad-token').default;
 
 const getAccessToken = async () => {
   const { CLIENT_SECRET, CLIENT_ID, DEFAULT_MS_GRAPH_SCOPE, MS_LOGIN_HOST_NAME, TENANT_ID } =
     process.env;
   try {
     const URL = `${MS_LOGIN_HOST_NAME}/${TENANT_ID}/oauth2/v2.0/token`;
-    const {
-      data: { access_token }
```

Comment:
try const {
      data: { access_token: accessToken }
    }

Reply:
It's not very readable. Sticking with what I have.